exchange/exchangetest/request-multi-bidders-debug-info.json

```diff
@@ -65,6 +65,7 @@
           {
             "uri": "appnexusTest.com",
             "requestbody": "appnexusTestRequestBody",
+            "requestheaders": null,
```

Comment:
Can we create a test were `"requestheaders"` is not null?

Reply:
@guscarreon : I have modified _request-multi-bidders-debug-info.json_ as per https://github.com/prebid/prebid-server/pull/1659/commits/92d3b49c3eb18db8e14e30743629b2a4d281b8ba 

_Changes_ 

1. Modified outgoing mock request for appnexus, to send some request header information.

2.Modified sample mock response such that `ext.debug.httpcalls.appnexus.requestheaders` will return the information of passed request headers